.travis.yml

```diff
@@ -14,6 +18,11 @@ addons:
     - gfortran
     - libopenblas-dev
     - liblapack-dev
+  homebrew:
```

Comment:
switched to running homebrew with this commands to fix caching (dropped roughly 2 minutes from the OSX build time)